src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsSettings/Properties/Resources.resx

```diff
@@ -1767,7 +1767,11 @@
     <value>Windows search</value>
     <comment>Area Cortana</comment>
   </data>
-  <data name="WindowsSecurity" xml:space="preserve">
+  <data name="WindowsSearchSettings" xml:space="preserve">
+    <value>Windows search settings</value>
+    <comment>Area Cortana</comment>
```

Comment:
What's "Area Cortana" here?

Reply:
> What's "Area Cortana" here?

This goes back to here, https://github.com/search?q=hash%3Ac9c54b7780dec2f55732fed053206ff7dadbcc0c not mine. But if you are asking me, "Cortana" was synonymous with "Search" for a while. And this is just the comment on the resource. I don't think that shows anywhere. Although this one does:

```
<data name="AreaCortana" xml:space="preserve">
    <value>Cortana</value>
  </data>
```
I think this might be something that should be changed out for "Search", but also could depend on what people want to see on different versions of Windows.